spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -224,6 +224,8 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			if (appDeploymentProperties.containsKey(INSTANCE_COUNT_PROPERTY_KEY)) {
```

Comment:
I will push a commit to my branch that makes the change I mentioned in the previous comment.